package com.findme.findme.Exceptions;

public class RelationshipAlreadyExistException extends RuntimeException{
}
